site/en/tutorials/text/text_classification_rnn.ipynb

```diff
@@ -554,13 +555,13 @@
       },
       "outputs": [],
       "source": [
-        "plt.figure(figsize=(16,8))\n",
-        "plt.subplot(1,2,1)\n",
+        "plt.figure(figsize=(16, 8))\n",
+        "plt.subplot(121)\n",
```

Comment:
nit: I find these a little too magical. Can we stick with `1, 2, 1`.

Reply:
Yep, too more syntactic sugar 😉